### ISSUE TITLE
Missing text in CodeSandbox sample

### ISSUE THREAD

**Issue by lbittner-pdftron** [developit/htm#49]
Hello! 

I was playing around with your CodeSandbox sample and noticed some missing text.

Here's my fork of the sample: https://codesandbox.io/s/7jx016lwo6

Notice that the `name` prop passed to header has some static text appended to the beginning.

```
<${Header} name="THIS SHOULD BE RENDERED (${page})" />
```

When the header is rendered, the static text is missing and just `All List` is rendered.

I might be missing something here but I would expect the header to say `THIS SHOULD BE RENDERED (All) List` instead of just `All List`.

Thanks!

EDIT: This is reproducible in your CodeSandbox demo as well - I just made a few changes to make it more clear.

**Comment by developit** [comment]
Looks like we're missing a test for this. Technically the correct syntax would be:

```js
html`<${Header} name=${'THIS SHOULD BE RENDERED (' + page + ')'} />`

```

**Comment by developit** [comment]
This is fixed in HTM 2.2 (codesandbox auto-updated!).
